### PR TITLE
[FEATURE] Add Referrer-Policy template

### PR DESCRIPTION
**Pull request by creopard**
This adds a new template for the Referrer-Policy in src/security

### CODE REVIEW

**Comment by S1SYPHOS** [comment]
:+1: 

Any progress on this?

File: src/security/referrer-policy.conf [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# This can be done by setting a `Referrer Policy` which
+# whitelists trusted sources of content for your website.
+#
+# To check your referrer policy, you can use an online service
+# such as: https://securityheaders.io/.
+#
+# https://scotthelme.co.uk/a-new-security-header-referrer-policy/
+# https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy
+#
+# Syntax:
```
Comment (LeoColomb) [inline src/security/referrer-policy.conf]: Thanks a lot for this really good doc-bloc!
(I'm just not sure if syntax examples are necessary, as they can change in the time and avoid user actully read the docs, right?)

File: src/security/referrer-policy.conf [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    # This should be the user agent's default behavior if no policy is
+    # specified.The origin is sent as referrer to a-priori as-much-secure
+    # destination (HTTPS->HTTPS), but isn't sent to a less secure destination
+    # (HTTPS->HTTP).
+
+    Header set Referrer-Policy: no-referrer-when-downgrade
+
+    # `mod_headers` cannot match based on the content-type, however,
+    # the `Content-Security-Policy` response header should be send
+    # only for HTML documents and not for the other resources.
```
Comment (LeoColomb) [inline src/security/referrer-policy.conf]: `Content-Security-Policy` oops! 😄

File: src/security/referrer-policy.conf [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+<IfModule mod_headers.c>
+
+    # no-referrer-when-downgrade (default)
+    # This should be the user agent's default behavior if no policy is
+    # specified.The origin is sent as referrer to a-priori as-much-secure
+    # destination (HTTPS->HTTPS), but isn't sent to a less secure destination
+    # (HTTPS->HTTP).
+
+    Header set Referrer-Policy: no-referrer-when-downgrade
```
Comment (LeoColomb) [inline src/security/referrer-policy.conf]: The colon is not needed.
```apacheconf
Header set Referrer-Policy no-referrer-when-downgrade
```

**Comment by creopard** [comment]
Please review the requested changes to referrer-policy.conf again.

**Pull request merged**